requirements.txt

```diff
@@ -1,4 +1,3 @@
-couleur==0.5.0
```

Comment:
You should include six in the `requirements.txt`, it doesn't come by default with Python 2.

Unrelated to this PR, but I'd make the other requirements more relaxed instead of pinning specific versions (maybe try minimum versions, such as `mypackage>=1.2.3`.